walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,14 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
```

Comment:
is this part of the commit or was it just an example?

Reply:
It was an example. I accidentally modified it (probably added a space somewhere) and added it to my commit. You can just ignore it. I shouldn't have pushed it to honest.